app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    try:
+        cmd, file, path = command.split()
+    except ValueError:
+        print("Command should have 2 arguments")
+        return
+
+    if cmd != "mv":
+        return
+
+    dirs = "/".join(path.split("/")[:-1])
```

Comment:
Don't use `/` to work with paths. Use functions from os module: os.path.join and os.path.split